app/src/main/java/org/simple/clinic/di/AppComponent.kt

```diff
@@ -22,10 +23,10 @@ interface AppComponent {
   fun inject(target: OtpSmsReceiver)
   fun inject(target: DataSync)
   fun inject(target: BottomSheetActivity)
-  fun inject(target: PatientFacilityChangeActivity)
 
   fun theActivityComponentBuilder(): TheActivityComponent.Builder
   fun setupActivityComponentBuilder(): SetupActivityComponent.Builder
+  fun patientFacilityChangeComponentBuilder() : PatientFacilityChangeComponent.Builder
```

Comment:
Can we rename this to retain the convention of the rest of the components?

`PatientFacilityChangeComponent` -> `PatientFacilityChangeActivityComponent`?
`patientFacilityChangeComponentBuilder` -> `patientFacilityChangeActivityComponentBuilder`

Reply:
I wanted to reduce the number of words here. I can rename it this way I guess.

Reply:
IMO, it's fine to be more verbose. This is something that won't get typed very often anyway/